### ISSUE TITLE
Phantom users finds no records when it should

### ISSUE THREAD

**Issue by walterrowe** [glFusion/glfusion#371]
I have several users who registered but never logged in. Batch Admin > Phantom users finds no records. I’m running 1.7.8.pl1.

Screenshot showing version
![3D609341-A4AC-4B2D-A50F-2A129CA2AC0C](https://user-images.githubusercontent.com/7131288/84961526-948b4f80-b0d2-11ea-9392-2d3321d4dbbe.png)

Screenshot showing users who registered but never logged in
![645581B2-584E-4662-BC69-E80B27967BA6](https://user-images.githubusercontent.com/7131288/84961525-93f2b900-b0d2-11ea-9a52-f7b97b841b50.png)

Screenshot showing Phantom users report with no records.
![4D3D7A79-B084-42E5-B705-88AB0B7B55CD](https://user-images.githubusercontent.com/7131288/84961523-935a2280-b0d2-11ea-8037-c6cac6f20161.png)

**Comment by mark0263** [comment]
I've looked at the SQL and verified it is correct on a couple of test systems and sample data. I'm unable to re-create the issue.

Question - it looks like the input fields in your screen shots are bit smaller than what I see on the default CMS theme - is it possible there is a larger number than 2 in the months field (i.e.; 22).  If you change it to 1 and hit Update List - does it show any records?

**Comment by walterrowe** [comment]
I changed it to 0 and it still came up empty. At least I think I did. Let me try again.

**Comment by mark0263** [comment]
Very odd - I'll keep looking to see if I can duplicate / determine the issue.

**Comment by walterrowe** [comment]
Just to be sure I overwrote the entire code with 1.7.8.pl1, cleared the cache, ran dvlpupdate, and tried it again. Same result. Entirely possible it is my site or data and not systemic. Just odd.
